nifi-toolkit/nifi-toolkit-cli/src/main/java/org/apache/nifi/toolkit/cli/impl/client/nifi/impl/JerseyFlowClient.java

```diff
@@ -217,4 +217,12 @@ public ActivateControllerServicesEntity activateControllerServices(final Activat
                     ActivateControllerServicesEntity.class);
         });
     }
+
+    @Override
+    public ClusteSummaryEntity getClusterSummary() throws NiFiClientException, IOException {
```

Comment:
Same import issue as above, also I assume the message should be "Error retrieving cluster summary" or something like that